make error on macosx 10.15.2

(base) Noahs-MacBook-Pro-2:vcftools noah$ make
make  all-recursive
make[1]: Entering directory '/Users/noah/vcftools'
Making all in src
make[2]: Entering directory '/Users/noah/vcftools/src'
Making all in cpp
make[3]: Entering directory '/Users/noah/vcftools/src/cpp'
clang -DHAVE_CONFIG_H -I. -I../..   -D_FORTIFY_SOURCE=2 -mmacosx-version-min=10.9  -march=core2 -mtune=haswell -mssse3 -ftree-vectorize -fPIC -fPIE -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -std=c++14 -fmessage-length=0 -MT vcftools-entry.o -MD -MP -MF .deps/vcftools-entry.Tpo -c -o vcftools-entry.o `test -f 'entry.cpp' || echo './'`entry.cpp
entry.cpp:32:34: error: use of undeclared identifier 'rpl_malloc'
        double * het_probs = (double *) malloc((size_t) (rare_copies + 1) * sizeof(double));
                                        ^
../../config.h:170:16: note: expanded from macro 'malloc'
#define malloc rpl_malloc
               ^
1 error generated.
make[3]: *** [Makefile:616: vcftools-entry.o] Error 1
make[3]: Leaving directory '/Users/noah/vcftools/src/cpp'
make[2]: *** [Makefile:319: all-recursive] Error 1
make[2]: Leaving directory '/Users/noah/vcftools/src'
make[1]: *** [Makefile:362: all-recursive] Error 1
make[1]: Leaving directory '/Users/noah/vcftools'
make: *** [Makefile:303: all] Error 2
(base) Noahs-MacBook-Pro-2:vcftools noah$

--- a/configure.ac
+++ b/configure.ac
@@ -63,9 +63,7 @@ AC_SYS_LARGEFILE
 
 # Checks for library functions.
 AC_FUNC_ERROR_AT_LINE
-AC_FUNC_MALLOC
-AC_FUNC_REALLOC
-AC_CHECK_FUNCS([gethostbyaddr gethostbyname memset pow select socket sqrt strchr strdup strerror strstr strtol])
+AC_CHECK_FUNCS([gethostbyaddr gethostbyname malloc memset pow realloc select socket sqrt strchr strdup strerror strstr strtol])
 
 # Optional features.
 AC_ARG_ENABLE([pca],

